src/CommonDBTM.php

```diff
@@ -6422,6 +6422,7 @@ public static function displayFullPageForItem(
         if (isset($menus[Session::getCurrentInterface()])) {
             // Load specific menus for this interface
             $menus = $menus[Session::getCurrentInterface()];
+            $menus = is_array($menus) ? $menus : [];
```

Comment:
Prevents following error when fix on front/ is not present:
```
[2023-02-22 07:32:22] glpiphplog.CRITICAL:   *** Uncaught Exception TypeError: CommonDBTM::displayAccessDeniedPage(): Argument #1 ($menus) must be of type array, string given, called in /var/www/webapps/glpi/src/CommonDBTM.php on line 6446 in /var/www/webapps/glpi/src/CommonDBTM.php at line 6342
  Backtrace :
  src/CommonDBTM.php:6446                            CommonDBTM::displayAccessDeniedPage()
  front/networkport.form.php:186                     CommonDBTM::displayFullPageForItem()
```

Reply:
This code is supposed to be able to detect pages which have different menu for central or helpdesk using specifics array keys:
```php
$menus = [
    'helpdesk' => [...],
    'central'    => [...]
]
```

In your care, you are not logged in so `Session::getCurrentInterface()` return false, which is converted to 0 and makes the condition true as $menus[0] will in fact be set.

I don't think we want to get into the condition at all in this case, as we are not trying to load specific helpdesk/central menus.

Maybe something like this would be clearer ?
```php
$interface = Session::getCurrentInterface();
if ($interface !== false && isset($menus[$interface])) {
    // Load specific menus for this interface
    $menus = $menus[$interface ];
}
```